modules/oc-by-example-content.adoc

```diff
@@ -1867,6 +1902,21 @@ Update fields of a resource
   
   # Update a container's image using a JSON patch with positional arrays
   oc patch pod valid-pod --type='json' -p='[{"op": "replace", "path": "/spec/containers/0/image", "value":"new image"}]'
+  
+  # Update a deployment's replicas through the scale subresource using a merge patch.
+  oc patch deployment nginx-deployment --subresource='scale' --type='merge' -p '{"spec":{"replicas":2}}'
+----
+
+
+
+== oc plugin list
+List all visible plugin executables on a user's PATH
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # List all available plugins
+  kubectl plugin list
```

Comment:
@soltysh This is being added with `kubectl` instead of `oc`. Isn't there somewhere where we transform all of the `kubectl`'s in the example usage to `oc`? Can we get this fixed in time for 4.11? (let me know if you need me to open a bug)